UnityProject/Assets/Scripts/Health/RespiratoryHealth/RespiratorySystem.cs

```diff
@@ -113,13 +113,17 @@ private bool Breathe(IGasMixContainer node)
 
 		GasMix gasMix = container.GasMix;
 		GasMix breathGasMix = gasMix.RemoveVolume(AtmosConstants.BREATH_VOLUME, true);
+		Vector3Int tileWorldPosition = gameObject.TileWorldPosition().To3Int();
+		Vector3Int position = transform.localPosition.RoundToInt();
+		MetaDataLayer metaDataLayer = MatrixManager.AtPoint(tileWorldPosition, true).MetaDataLayer;
 
 		float oxygenUsed = HandleBreathing(breathGasMix);
 
 		if (oxygenUsed > 0)
 		{
 			breathGasMix.RemoveGas(Gas.Oxygen, oxygenUsed);
 			node.GasMix.AddGas(Gas.CarbonDioxide, oxygenUsed);
+			metaDataLayer.UpdateSystemsAt(position);
```

Comment:
one-line and more reliable solution to what you're trying to achieve would be:
```suggestion
			MatrixManager.Get(playerScript.PlayerSync.ServerState.MatrixId).MetaDataLayer.UpdateSystemsAt(playerScript.PlayerSync.ServerLocalPosition);

```
so it'd be taking current matrix info from player's state.
also please don't use `transform.position` or `transform.localPosition` unless you're working on clientside lerping and such:
![img](https://user-images.githubusercontent.com/10403536/74655717-ba383480-5184-11ea-9cc1-d820173c6549.png)

Reply:
Fixed.